docs/plugins/find-and-install-plugin.md

```diff
@@ -0,0 +1,24 @@
+(find-and-install-plugins)=
+
+# Finding and installing a napari plugin
+
+You can find plugins to use with napari on the [napari hub](https://napari.org),
+and can install plugins directly from your instance of napari itself.
+
+## Finding plugins on the napari hub
+
+The napari hub hosts information about all napari-compatible plugins.
+You can browse, search, and filter to find plugins that fit your needs.
+You can also share links to specific search results and plugins.
+
+## Installing plugins with napari
+
+All plugins listed on the napari hub can be installed directly from napari:
+
+1. From the “Plugins” menu, select “Install/Uninstall package(s)...”.
```

Comment:
I think we renamed this “Install/Uninstall plugins...” in #3071

Can you update with the latest screenshot that will be out in the next release

<img width="270" alt="Screen Shot 2021-07-27 at 12 10 10 PM" src="https://user-images.githubusercontent.com/6531703/127213858-fa38c5ed-24e1-4e32-a7c3-43d5826cd14f.png">